electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -69,8 +76,50 @@ def append(
         if event:
             self.events.append(event)
 
+    @staticmethod
+    def merge_production_breakdowns(
+        ungrouped_production_breakdowns: List["ProductionBreakdownList"],
+        logger: Logger,
+        merge_zone_key: ZoneKey,
+        merge_source: str,
```

Comment:
Instead of specifying the new source here maybe we should sort the existing soures and format them into the following format if they are diffrent and deduplicate them if not:
`source1, source2` I think that is already in use by some parsers and would make the experience consistant and would also allow us to change to a list later on when all the parsers use the new class in a easy manner. (We have talked about trying to make this a list or array for the frontend as well.)

Example from the SG parser:
```python
    return {
        "datetime": sg_data_to_datetime(data),
        "zoneKey": zone_key,
        "production": generation_by_type,
        "storage": {},  # there is no known electricity storage in Singapore
        "source": "emcsg.com, ema.gov.sg",
    }
```

Reply:
Agree this might be better, as I saw some parsers do that like the India parser. Initially I thought, better to have the user in control of the final source to avoid any surprises but this might be better.